src/main/java/com/cosc436002fitzarr/brm/models/buildingriskassessment/BuildingProfile.java

```diff
@@ -0,0 +1,64 @@
+package com.cosc436002fitzarr.brm.models.buildingriskassessment;
+
+import com.cosc436002fitzarr.brm.enums.RiskLevel;
+import java.util.Objects;
+
+public class BuildingProfile {
+    private String buildingName;
```

Comment:
We could probably add another enum here, something like `RiskProbability` which the site maintenance associate will fill out on a form.